src/pstress.cpp

```diff
@@ -19,6 +19,11 @@
 #include <INIReader.hpp>
 #include <mysql.h>
 #include <thread>
+#include <string>
+#include <libgen.h>
```

Comment:
Please mention the purpose of inclusion. This can help us later, during refactoring or removing the unnecessary header files in future.
```suggestion
#include <libgen.h> // dirname()
```